src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -427,13 +428,36 @@ private void updateSupplementaryEvidence(
 
         if (attachedToCase.isPresent()) {
```

Comment:
of this is present it means it is updated and status already changed.
if it is present we should trow an exception.